packages/wallet/src/SingleAddressWallet.ts

```diff
@@ -402,7 +402,7 @@ export class SingleAddressWallet implements ObservableWallet {
     this.#logger.debug(`Submitting transaction ${tx.id}`);
     this.#newTransactions.submitting$.next(tx);
     try {
-      await this.txSubmitProvider.submitTx(coreToCsl.tx(tx).to_bytes());
+      await this.txSubmitProvider.submitTx({ signedTransaction: coreToCsl.tx(tx).to_bytes() });
```

Comment:
Need to pass hex string here too

Reply:
addressed here: https://github.com/input-output-hk/cardano-js-sdk/pull/393/commits/733679540e91c06b03b3a195cd1e5a352011a9df